.circleci/config.yml

```diff
@@ -274,6 +292,14 @@ workflows:
           filters:
             branches:
               ignore: /^pull\/.*/
+  pointers:
+    jobs:
+      - tweet-release:
+          filters:
+            tags:
+              only: /v.*/
```

Comment:
It looks like the tagged releases start with a v so I just want to clarify that you want the v added to the filter? Otherwise, tagged releases won't trigger the tweet.

`^v[0-9]+\.[0-9]+\.[0-9]+`

![image](https://user-images.githubusercontent.com/8462764/196047588-3d073bba-8ff6-4338-a43e-50c52fbcba72.png)

Reply:
I'm actually wondering if we should use the filter that publish workflow jobs are using:

`/^v([0-9]+)\.([0-9]+)\.([0-9]+)(?:([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+[0-9A-Za-z-]+)?$/`

Reply:
yes, you are correct @ericabrauer.  we should set it to the same as the other publish jobs.